owncloudApp/src/main/java/com/owncloud/android/presentation/files/operations/FileOperationsViewModel.kt

```diff
@@ -95,6 +99,8 @@ class FileOperationsViewModel(
 
     val openDialogs = mutableListOf<FileAlreadyExistsDialog>()
 
+    private val _deepLinkFlow = MutableStateFlow<Event<UIResult<OCFile?>>?>(null)
+    val deepLinkFlow: StateFlow<Event<UIResult<OCFile?>>?> = _deepLinkFlow
```

Comment:
Place these just after the other LiveDatas and Flows

Reply:
Done